test/index.spec.mjs

```diff
@@ -2,6 +2,9 @@ import assert from 'node:assert';
 import { resolve } from 'node:path';
 import { createRequire } from 'module';
 import dedupe from '../build/dedupe.mjs';
+import readJson from '../build/readJson.mjs';
+
+const masterableCategories = await readJson(new URL('../config/masterableCategories.json', import.meta.url));
```

Comment:
we already have a bypass as part of the main load for doing that, should just use the require we already have, as that will read the json... (i.e. don't need `readJson`)